packages/jaeger-ui/src/components/TracePage/TraceStatistics/generateDropdownValue.tsx

```diff
@@ -33,53 +36,51 @@ function getValueTagIsPicked(tableValue: ITableSpan[], trace: Trace, nameSelecto
       for (let j = 0; j < allSpans.length; j++) {
         for (let l = 0; l < allSpans[j].tags.length; l++) {
           if (nameSelectorTitle === allSpans[j].tags[l].key) {
-            availableTags.push(allSpans[j]);
+            spansWithFilterTag.push(allSpans[j]);
           }
         }
       }
     }
   }
-  availableTags = [...new Set(availableTags)];
+  spansWithFilterTag = [...new Set(spansWithFilterTag)];
 
-  const tags = _(availableTags).map('tags').flatten().value();
-  let tagKeys = _(tags).map('key').uniq().value();
-  tagKeys = _.filter(tagKeys, function calc(o) {
-    return o !== nameSelectorTitle;
-  });
-  availableTags = [];
-  availableTags.push(serviceName);
-  availableTags.push(operationName);
-  availableTags = availableTags.concat(tagKeys);
+  const tags = spansWithFilterTag.map(o => o.tags);
+  let tagKeys = _uniq(_map(_flatten(tags), 'key'));
+  tagKeys = tagKeys.filter(o => o !== nameSelectorTitle);
+  spansWithFilterTag = [];
```

Comment:
these are not spans anymore, this is just output. Could this not be simplified like this?
```javascript
return [serviceName, operationName, ... tagKeys];
```

Reply:
or `_concat()`

Reply:
Used the Spread syntax, to remove `_concat` completely.